packages/epics/src/people/components/edit-person-head.tsx

```diff
@@ -25,7 +33,7 @@ export const EditPersonHead = ({
       >
         <Image
           className="rounded-lg mr-3"
-          src={avatar ?? ''}
+          src={avatar || '/placeholder/space-avatar-image.png'}
```

Comment:
Let's always handle those in the props section. This gives the most visibility, imo